securedrop/tests/migrations/migration_2d0ce3ee5bdc.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+
+import random
+import string
+
+from sqlalchemy import text
+
+from db import db
+from journalist_app import create_app
+from .helpers import (random_bool, random_chars, random_username, random_bytes,
+                      random_datetime, bool_or_none)
+
+random.seed('ᕕ( ᐛ )ᕗ')
+
+
+class Helper():
+
+    @staticmethod
+    def add_source():
+        filesystem_id = random_chars(96) if random_bool() else None
+        params = {
+            'filesystem_id': filesystem_id,
+            'journalist_designation': random_chars(50),
+            'flagged': bool_or_none(),
+            'last_updated': random_datetime(nullable=True),
+            'pending': bool_or_none(),
+            'interaction_count': random.randint(0, 1000),
```

Comment:
and we'll need to add the UUID column here otherwise the unique constraint on UUID will cause an integrity error if UUID is null everywhere in the sources table